Lib/test/test_tarfile.py

```diff
@@ -3056,6 +3057,69 @@ def test_keyword_only(self, mock_geteuid):
                               tarfl.extract, filename_1, TEMPDIR, False, True)
 
 
+@os_helper.skip_unless_working_chmod
+class ExtractStickyFileTest(unittest.TestCase):
+
+    # to use in mock tests, on platforms that don't have it
+    fake_EFTYPE = getattr(errno, "EFTYPE", max(errno.errorcode) + 100)
+
+    def setUp(self):
+        self.test_dir = os.path.join(TEMPDIR, "chmod")
+        os.mkdir(self.test_dir)
+        self.addCleanup(os_helper.rmtree, self.test_dir)
+        self.sticky_mode = "-rwxrwxrwt"
+        self.non_sticky_mode = "-rwxrwxrwx"
+
+    def extract_sticky_file(self, file_name="sticky"):
+        with ArchiveMaker() as arc:
+            arc.add(file_name, mode=self.sticky_mode)
+        with arc.open(errorlevel=2) as tar:
+            tar.extract(file_name, self.test_dir, filter="fully_trusted")
+        self.extracted_path = os.path.join(self.test_dir, file_name)
+
+    def test_extract_chmod_eftype_success(self):
```

Comment:
On most platforms, this test willl be run without EFTYPE. I suggest to remove it:

```suggestion
    def test_extract_chmod_success(self):
```